tests/realtikvtest/addindextest3/operator_test.go

```diff
@@ -130,7 +130,6 @@ func TestBackfillOperators(t *testing.T) {
 		results := sink.Collect()
 		cnt := 0
 		for _, rs := range results {
-			require.NoError(t, rs.Err)
```

Comment:
why remove it

Reply:
It is checked nowhere. I chose to use `w.ctx.onError()` to handle error and to terminate the pipeline from the beginning operators.